sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -700,6 +700,9 @@ def __init__(self, n):
     # helper instead.
     self._top_combiner = TopCombineFn(n)
 
+  def setup(self):
+    self._top_combiner.setup()
```

Comment:
I think this would fail the Dataflow V1 check as well.
I thought of possible improvement of the check and this call would cause an issue.

[`bad0a52` (#13267)](https://github.com/apache/beam/pull/13267/commits/bad0a522ced25cc6ceef3ad14563ed2f6419788f)